homeassistant/components/binary_sensor/deconz.py

```diff
@@ -29,6 +30,7 @@ def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
         if sensor and sensor.type in DECONZ_BINARY_SENSOR:
             entities.append(DeconzBinarySensor(sensor))
     async_add_devices(entities, True)
+    hass.data[DECONZ_ENTITIES] = hass.data[DECONZ_ENTITIES] + entities
```

Comment:
This is not allowed. Main component should not have access to the entity objects.

Reply:
I guess I could just do a translation dict between entity name and deconz id instead